starter-repos/app-template/.gitlab-ci.yml

```diff
@@ -16,6 +16,12 @@
 include:
 - project: 'platform-admins/shared-ci-cd'
   file: 'ci/go-docker.yaml'
+- project: 'platform-admins/shared-ci-cd'
+  file: 'skaffold/build.yaml'
+- project: 'platform-admins/app-template'
```

Comment:
A local file can be simply as
- local 'platform-admins/app-template'

Reply:
Thanks! Chose the `local: /` style.